lib/simp/cli/utils.rb

```diff
@@ -30,6 +30,11 @@ class PuppetInfo
     attr_reader :system_puppet_info
 
     def initialize(environment)
+      require 'puppet'
+      require 'facter'
```

Comment:
shouldn't the `facter` annd `puppet` requires be in `load_custom_facts` instead?